docs/conf.py

```diff
@@ -32,10 +32,13 @@
     "sphinx.ext.napoleon",
     "IPython.sphinxext.ipython_console_highlighting",
     #"myst_nb",
+    "myst_parser",
     "nbsphinx"
 
 ]
 
+source_suffix = [".rst", ".md"]
```

Comment:
Do not add `source_suffix` to conf.py because it causes error in Sphinx build, see commits under this comment:
https://github.com/r5py/r5py/pull/38#pullrequestreview-969167018

Reply:
My bad. We do seem to need this because `myst_nb` is not used.